src/main/java/billgates/view/BillGatesUtilities.java

```diff
@@ -0,0 +1,42 @@
+package billgates.view;
+
+import java.util.*;
+
+public class BillGatesUtilities {
+
+    public static final String DATETIME_PATTERN = "yyyy-MM-dd HH:mm:ss";
+    public static final String TIME_PATTERN = "HH:mm:ss";
+
+    static {
+        CURRENCY_CODES = getCurrencyCodes();
+    }
+
+    /**
+     * A list of all available currency codes of the current JVM.
+     * It starts with CAD, CNY, and USD; and follows by all currency codes sorted by ascending
+     * alphabetical order.
+     */
+    public static final List<String> CURRENCY_CODES;
+
+    /**
+     * Gets a list of available currency codes according to ISO 3166.
+     * Notes: CAD, CNY, USD are the top three currencies.
+     *
+     * @return a list of currency codes sorted in ascending alphabetical order with CAD, CNY,
+     * and USD at the beginning.
+     */
+    public static List<String> getCurrencyCodes() {
```

Comment:
Woah this is actually pretty cool!

Reply:
Small issue tho: 

![image](https://user-images.githubusercontent.com/8016180/205576175-339959fc-f9e4-47b0-a2b6-e916e0156d71.png)

Maybe we want to only include the `CurrencyCode` if it is not in `temp`.

Reply:
Fixed in #96